src/libraries/System.IO.FileSystem/src/System/IO/FileSystem.Unix.cs

```diff
@@ -129,7 +82,7 @@ public static void ReplaceFile(string sourceFullPath, string destFullPath, strin
             }
 
             // Finally, rename the source to the destination, overwriting the destination.
-            Interop.CheckIo(Interop.Sys.Rename(sourceFullPath, destFullPath));
+            Interop.CheckIo(Interop.Sys.Rename(sourceFullPath, destFullPath, 1));
         }
 
         public static void MoveFile(string sourceFullPath, string destFullPath)
```

Comment:
Is the TICTOU bug also possible to reproduce on `File.Move` and `File.Replace`? I am trying to understand why these methods were also modified.

Reply:
Yes. See PR title.